commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -72,8 +72,9 @@
 	QUALITY_MEASURE_SECTION_REQUIRED_REPORTING_PARAM_REQUIREMENT(32, "The Quality Measure Section must have "
 			+ "exactly one Reporting Parameter ACT"),
 	PERFORMANCE_RATE_INVALID_VALUE(33, "Must enter a valid Performance Rate value"),
-	CPC_CLINICAL_DOCUMENT_MISSING_PRACTICE_SITE_ADDRESS(34, "Must contain a practice site address for CPC+ "
-			+ "conversions"),
+	CPC_CLINICAL_DOCUMENT_MISSING_PRACTICE_SITE_ADDRESS(34, "CPC+ submissions must contain a practice site address."
+		+ " Please refer to the `(Submission year's)` IG for more details: " + DocumentationReference.PRACTICE_SITE_ADDRESS
```

Comment:
Perhaps not a colon for after `details` since the sentence continues after the URL.